pkg/controller/configmap/config.go

```diff
@@ -339,6 +354,15 @@ func inSlice(str string, s []string) bool {
 	return false
 }
 
+func inMapKey(str string, m map[string][]string) bool {
```

Comment:
I might be wrong but perhaps there's an easier way for this?

_val, found := m[key]

source: https://blog.golang.org/maps

Reply:
n_n👍